client/app/bundles/course/survey/containers/SurveyFormDialogue/SurveyForm.jsx

```diff
@@ -68,158 +72,238 @@ const surveyFormTranslations = defineMessages({
   },
 });
 
-const validate = (values) => {
-  const errors = {};
+const validationSchema = yup.object({
+  title: yup.string().required(formTranslations.required),
+  description: yup.string(),
+  start_at: yup.date().nullable().required(formTranslations.required),
+  end_at: yup
+    .date()
+    .nullable()
+    .min(yup.ref('start_at'), translations.startEndValidationError)
+    .when('allow_response_after_end', {
+      is: true,
+      then: yup
+        .date()
+        .min(yup.ref('start_at'), translations.startEndValidationError)
+        .typeError(formTranslations.required)
+        .required(formTranslations.required),
+    }),
+  base_exp: yup
```

Comment:
Previously this wasn't required right? Not an issue, I think it makes sense to add it.

Reply:
There is a backend check but no frontend check. I am adding here so that the validation is done at frontend.